templates/terraform/examples/ha_vpn_gateway_basic.tf.erb

```diff
@@ -0,0 +1,205 @@
+resource "google_compute_ha_vpn_gateway" "ha_gateway1" {
```

Comment:
I'd recommend having two test examples: the basic one would be "can we create a gateway at all?" (which will probably fail faster if the resource is broken and also won't fail at all if one of the later resources is broken), and then this one as a more descriptive example (so instead of calling it ha_vpn_gateway_basic, maybe something like ha_vpn_gateway_gcp_to_gcp)